### ISSUE TITLE
Add Lambda request id 

### ISSUE THREAD

**Issue by benkehoe** [awslabs/aws-embedded-metrics-python#39]
The `aws_request_id` field from the context object should get added to the default set of properties. https://docs.aws.amazon.com/lambda/latest/dg/python-context.html

**Comment by jaredcnance** [comment]
The library doesn’t have access to the context object. Are you asking for a method on the logger that would allow you to add the context? Or are you wanting a lambda_metric_scope that wraps your handler and adds the request id automatically for you?

**Comment by benkehoe** [comment]
I mean, in an ideal world the Lambda Python runtime would put the request id in the [environment variables](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html#configuration-envvars-runtime), since we're [already using that to populate the context](https://github.com/awslabs/aws-embedded-metrics-python/blob/c85dcdd9a45be18aba7ffbcbd0e540a7a88d55f7/aws_embedded_metrics/environment/lambda_environment.py#L43).

I guess it could be a separate `lambda_metric_scope` decorator, though it seems like in `metric_scope` if the environment says it's type `AWS::Lambda::Function` then you'd know to find the context in the arguments.

**Comment by jaredcnance** [comment]
Makes sense. I think it should be pretty straightforward to scrape the input arguments for the request id.
